x-pack/plugins/ml/public/application/overview/components/anomaly_detection_panel/utils.ts

```diff
@@ -158,5 +164,14 @@ export function getStatsBarData(jobsList: any) {
 
   jobStats.activeNodes.value = Object.keys(mlNodes).length;
 
+  if (jobStats.total.value === 0) {
+    for (const statKey in jobStats) {
+      if (statKey !== 'total') {
+        // @ts-ignore
```

Comment:
same with this `@ts-ignore`

Reply:
this one was removed in https://github.com/elastic/kibana/pull/159609/commits/0380da1f6d351e6c0e45f9ffccf398b5fafdbcf0